airflow/models/dagbag.py

```diff
@@ -201,18 +202,22 @@ def get_dag(self, dag_id, session: Session = None):
                 dag_id in self.dags_last_fetched
                 and timezone.utcnow() > self.dags_last_fetched[dag_id] + min_serialized_dag_fetch_secs
             ):
+                sd_latest_version = SerializedDagModel.get_latest_version_hash(dag_id=dag_id, session=session)
```

Comment:
Should (can) we cache this? One additional db query per call seems pretty significant.

Or maybe this can be merged with `get_last_updated_datetime` below.

Reply:
you're right. I merged the two queries into a single one in a new method. From a quick look around, `get_last_updated_datetime` does not seems to be used anymore, is it fine to remove it in this PR?